### PR TITLE
[xml] Update portuguese.xml

### PR DESCRIPTION
**Pull request by Blackspirits**
@hugok79
minor corrections.

### CODE REVIEW

**Comment by hugok79** [comment]
@donho @Blackspirits Looks good to me but the recent commits are missing... 

https://github.com/notepad-plus-plus/notepad-plus-plus/commit/d039eaa9009e3561589cb3b5e9849ab9aefa3e02
https://github.com/notepad-plus-plus/notepad-plus-plus/commit/78d0e7e12f3d164eec1b6b1851ad21f987b0dc6d
https://github.com/notepad-plus-plus/notepad-plus-plus/commit/4e2903a27461de29d3746db5a40cc77d6a2159d7
https://github.com/notepad-plus-plus/notepad-plus-plus/commit/4e6bbbc17ffbf765d095b63414d534066fe4c877
https://github.com/notepad-plus-plus/notepad-plus-plus/commit/e497ae2c069ab7a82a3cd7b0da52e48233060c57
https://github.com/notepad-plus-plus/notepad-plus-plus/commit/ffc0ed2516f9f848d648e9b1435a8c31b9254963
https://github.com/notepad-plus-plus/notepad-plus-plus/commit/2724e0ddbb1e7349fd9c23711d6b4e9535bcf1ba
https://github.com/notepad-plus-plus/notepad-plus-plus/commit/9bc790b014a7e0ea5b935d2ef963d02b6793cd2b
https://github.com/notepad-plus-plus/notepad-plus-plus/commit/05f339b0cfe11095dc5590241bc8ff8cc4aec835
https://github.com/notepad-plus-plus/notepad-plus-plus/commit/cecd161570eb22ef8bc6f730f725d81ec67fbf7e
https://github.com/notepad-plus-plus/notepad-plus-plus/commit/89278e1dc70d0171e1868cff359c38bbd0b46e00
https://github.com/notepad-plus-plus/notepad-plus-plus/commit/1c27e64126603672babe3489eff85407067e8e9b
https://github.com/notepad-plus-plus/notepad-plus-plus/commit/591b00e538052543376e937a220aa6ef024ed10b
https://github.com/notepad-plus-plus/notepad-plus-plus/commit/e30ee852d64e0c4cb8db676760b77b14d6be93f3
https://github.com/notepad-plus-plus/notepad-plus-plus/commit/39001d7a0272320124117e49fa7ac333a3a9d96e

I can make a new pull request to add those recente commits!

**Comment by Blackspirits** [comment]
@donho @hugok79 I've edited my pull with your commits and made a few more translation improvements

**Comment by hugok79** [comment]
@Blackspirits Ok. I’m going to close my pull request.
Thank you for your work and help

**Comment by hugok79** [comment]
@Blackspirits In line 227	**Item id="43009" name="Ir para a chaveta correspondida"/** should be before **Item id="43010" name="Localizar anterior"/**

File: PowerEditor/installer/nativeLang/portuguese.xml [hunk 1]
```diff
@@ -211,8 +223,8 @@ Translation note:
 					<Item id="43050" name="Inverter marcador"/>
 					<Item id="43052" name="Localizar carateres..."/>
 					<Item id="43053" name="Selecionar tudo entre as chavetas correspondidas"/>
-					<Item id="43009" name="Ir para a chaveta correspondida"/>
 					<Item id="43010" name="Localizar anterior"/>
+					<Item id="43009" name="Ir para a chaveta correspondida"/>
```
Comment (hugok79) [inline PowerEditor/installer/nativeLang/portuguese.xml:227]: id="43009" should be before id="43010"

Comment (Blackspirits) [reply PowerEditor/installer/nativeLang/portuguese.xml:226]: Done

Comment (hugok79) [reply PowerEditor/installer/nativeLang/portuguese.xml:226]: **id=43011** is not aligned with the text!

File: PowerEditor/installer/nativeLang/portuguese.xml [hunk 2]
```diff
@@ -212,8 +224,7 @@ Translation note:
 					<Item id="43052" name="Localizar carateres..."/>
 					<Item id="43053" name="Selecionar tudo entre as chavetas correspondidas"/>
 					<Item id="43009" name="Ir para a chaveta correspondida"/>
-					<Item id="43010" name="Localizar anterior"/>
-					<Item id="43011" name="Pesquisa &amp;progressiva"/>
+					<Item id="43010" name="Localizar anterior"/>										<Item id="43011" name="Pesquisa &amp;progressiva"/>
```
Comment (hugok79) [inline PowerEditor/installer/nativeLang/portuguese.xml:227]: id="43011" line is not aligned with the text

**Pull request closed without merging**